### PR TITLE
Add function to get active message hash for a given account address.

### PR DESCRIPTION
**Pull request by deepesh-kn**
- Added `getInboxActiveProcess()` function.
- Added `getOutboxActiveProcess()` function.
- Added corresponding test cases.

fixes: #595

### CODE REVIEW

File: contracts/test/gateway/TestGatewayBase.sol [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+     * @param _messageHash MessageHash for which status is the be set.
+     */
+    function setInboxProcess(
+        address _account,
+        uint256 _nonce,
+        bytes32 _messageHash
+    )
+        external
+    {
+        super.registerInboxProcess(_account, _nonce, _messageHash);
```
Comment (sarvesh-ost) [inline contracts/test/gateway/TestGatewayBase.sol]: Why it's important to have nonce in the argument? 
can we not directly set this?  
```
inboxActiveProcess[_account] = _messageHash;
```

Similar comment for setOutboxProcess

File: contracts/gateway/GatewayBase.sol [hunk 2]
```diff
@@ -338,20 +338,71 @@ contract GatewayBase is Organized {
         emit BountyChangeConfirmed(previousBountyAmount_, changedBountyAmount_);
     }
 
+    /**
+     * @notice Method to get the active message hash and its status from inbox
+     *         for the given account address.
+     *
+     * @dev If message hash do not exists for the given account address then
```
Comment (schemar) [inline contracts/gateway/GatewayBase.sol]:
```suggestion
     * @dev If message hash does not exist for the given account address then
```
Also, is this not relevant to the user of the function? Why in `@dev`?

File: contracts/gateway/GatewayBase.sol [hunk 3]
```diff
@@ -338,20 +338,71 @@ contract GatewayBase is Organized {
...
+    {
+        messageHash_ = inboxActiveProcess[_account];
+        status_ = messageBox.inbox[messageHash_];
+    }
+
+    /**
+     * @notice Method to get the active message hash and its status from outbox
+     *         for the given account address.
+     *
+     * @dev If message hash do not exists for the given account address then
```
Comment (schemar) [inline contracts/gateway/GatewayBase.sol]:
```suggestion
     * @dev If message hash does not exist for the given account address then
```
Also, is this not relevant to the user of the function? Why in @dev?

File: contracts/gateway/GatewayBase.sol (hunk 2 above)
Comment (deepesh-kn) [reply contracts/gateway/GatewayBase.sol]: Removed from the @dev and added to notice.
